src/components/Onfido/BaseOnfidoWeb.js

```diff
@@ -17,118 +17,135 @@ const propTypes = {
-                this.props.onUserExit(userExitCode);
-            },
-            onModalRequestClose: () => {
-                Log.hmmm('Onfido user closed the modal');
+            {
+                type: CONST.ONFIDO.TYPE.FACE,
+                options: {
+                    requestedVariant: CONST.ONFIDO.VARIANT.VIDEO,
+                    uploadFallback: false,
+                },
             },
-            language: {
-                // We need to use ES_ES as locale key because the key `ES` is not a valid config key for Onfido
-                locale: this.props.preferredLocale === CONST.LOCALES.ES ? CONST.LOCALES.ES_ES_ONFIDO : this.props.preferredLocale,
+        ],
+        smsNumberCountryCode: CONST.ONFIDO.SMS_NUMBER_COUNTRY_CODE.US,
+        showCountrySelection: false,
+        onComplete: (data) => {
+            if (_.isEmpty(data)) {
+                Log.warn('Onfido completed with no data');
+            }
+            onSuccess(data);
+        },
+        onError: (error) => {
+            const errorMessage = lodashGet(error, 'message', CONST.ERROR.UNKNOWN_ERROR);
+            const errorType = lodashGet(error, 'type');
+            Log.hmmm('Onfido error', {errorType, errorMessage});
+            onError(errorMessage);
+        },
+        onUserExit: (userExitCode) => {
+            Log.hmmm('Onfido user exits the flow', {userExitCode});
+            onUserExit(userExitCode);
+        },
+        onModalRequestClose: () => {
+            Log.hmmm('Onfido user closed the modal');
+        },
+        language: {
+            // We need to use ES_ES as locale key because the key `ES` is not a valid config key for Onfido
+            locale: preferredLocale === CONST.LOCALES.ES ? CONST.LOCALES.ES_ES_ONFIDO : preferredLocale,
 
-                // Provide a custom phrase for the back button so that the first letter is capitalized,
-                // and translate the phrase while we're at it. See the issue and documentation for more context.
-                // https://github.com/Expensify/App/issues/17244
-                // https://documentation.onfido.com/sdk/web/#custom-languages
-                phrases: {
-                    'generic.back': this.props.translate('common.back'),
-                },
+            // Provide a custom phrase for the back button so that the first letter is capitalized,
+            // and translate the phrase while we're at it. See the issue and documentation for more context.
+            // https://github.com/Expensify/App/issues/17244
+            // https://documentation.onfido.com/sdk/web/#custom-languages
+            phrases: {
+                'generic.back': translate('common.back'),
             },
-        });
+        },
+    });
+}
 
-        window.addEventListener('userAnalyticsEvent', (event) => {
-            Log.hmmm('Receiving Onfido analytic event', event.detail);
+const Onfido = forwardRef((props, ref) => {
```

Comment:
Asked a question about this in slack: https://expensify.slack.com/archives/C01GTK53T8Q/p1689360729357789

Reply:
Actually, I have dug into this some more and unless I'm missing something it appears unused, and don't need this ref at all:

1. Remove `forwardRef` from here
1. Don't pass the ref to the div on line 145
1. Remove the ref and `this.baseOnfido` from the parent, because it's not used anywhere.

Reply:
@roryabraham it's used here: https://github.com/Expensify/App/blob/0b00e1cc82a4f6023aff6b89796a91fba6be91a2/src/components/Onfido/index.website.js#L13

Reply:
Ah, you're right I missed that.